src/graph/calc.rs

```diff
@@ -210,8 +210,11 @@ fn update_commit_line<'a>(
     commit: &'a Commit,
     commit_line_state: &mut [Option<&'a CommitHash>],
     target_commit_hashes: &[&CommitHash],
-) -> u8 {
-    let mut min_pos_x = commit_line_state.len();
+) -> usize {
+    if commit_line_state.is_empty() {
+        return 0;
+    }
+    let mut min_pos_x = commit_line_state.len().saturating_sub(1);
     for target_hash in target_commit_hashes {
```

Comment:
It doesn't seem necessary for this PR.

Reply:
I've resolved this change back to the `master`.